frontend/src/v5/ui/components/dashboard/dashboardList/dashboardListItem/components/dashboardListItemTitle/dashboardListItemFederationTitle/dashboardListItemFederationTitle.component.tsx

```diff
@@ -0,0 +1,58 @@
+/**
+ *  Copyright (C) 2022 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { viewerRoute } from '@/v5/services/routing/routing';
+import { UploadStatuses } from '@/v5/store/containers/containers.types';
+import { IFederation } from '@/v5/store/federations/federations.types';
+import { RevisionStatus } from '@/v5/ui/routes/dashboard/projects/containers/containersList/latestRevision/revisionStatus';
+import { DashboardParams } from '@/v5/ui/routes/routes.constants';
+import { IFixedOrGrowContainer } from '@controls/fixedOrGrowContainer/fixedOrGrowContainer.component';
+import { Highlight } from '@controls/highlight';
+import { FormattedMessage } from 'react-intl';
+import { useParams } from 'react-router';
+import { Link } from 'react-router-dom';
+import { DashboardListItemTitle } from '../dashboardListItemTitle.component';
+
+interface IFederationTitle extends IFixedOrGrowContainer {
+	federation: IFederation;
+	filterQuery?: string;
+}
+
+export const DashboardListItemFederationTitle = ({
+	federation,
+	filterQuery = '',
+}: IFederationTitle): JSX.Element => {
+	const { teamspace, project } = useParams<DashboardParams>();
+
+	const { status, desc, name } = federation;
+	const subtitle = status === UploadStatuses.OK ? desc : <RevisionStatus status={status} name={name} />;
```

Comment:
I know this could  a bit picky but can we change the variable name subtitle to uploadStatus ? that whay when you take a look at just the JSX bit you already know what it is.
(I was just reading at the jsx code alone and I couldn't see how the status was being shown )